src/schema.graphqls

```
@@ -27,6 +27,9 @@ type Query {
 
   """ Returns all claims the member has """
   claims: [Claim!]!
+
+  """ Returns perils from promise-cms """
+  perilsV2(typeOfContract: TypeOfContract!, locale: Locale!): [PerilV2]
```

Should this not be typed like this?

```suggestion
  perilsV2(typeOfContract: TypeOfContract!, locale: Locale!): [PerilV2!]!
```

---

Does every type of contract exist for every locale? In that case yes :)

---

I guess we wont have `typeOfContract: SE_APARTMENT_BRF, locale: nb_NO`, but maybe we want to throw an error in that case? @robinandeer

---

Right, wouldn't those just be filtered out so we end up with an empty array? I think that would be okay to deal with on the frontend, no?

```json
{
	"perilsV2": []
}
```

I believe with this typing, a valid response could look like this:

```json
{
	"perilsV2": [{ ... }, null, null, { ... }]
}
```

...and we don't want to handle that every time hehe